app/controllers/events_controller.rb

```diff
@@ -143,7 +141,6 @@ def download_agreement_letters
     if not params.has_key?(:selected_participants)
       redirect_to event_participants_url(@event), notice: I18n.t('events.agreement_letters_download.notices.no_participants_selected') and return
     end
-    authorize! :print_agreement_letters, @event
```

Comment:
Same here - the check will now be `download_agreement_letters` instead of `print_agreement_letters`.